uit/gui_tools/submit.py

```diff
@@ -50,8 +51,10 @@ def pbs_options_view(self):
 class PbsScriptAdvancedInputs(HpcConfigurable):
     env_names = param.List()
     env_values = param.List()
+    env_browser = param.List()
     browse = param.Action(lambda self: self.toggle_file_browser(), label='📂')
     browse_toggle = param.Boolean(default=False)
```

Comment:
I believe we can remove both the `browse` and the `browse_toggle` parameters now, right? Are they being used anywhere?

Reply:
That is correct, and I have removed them in the newest version of the file.